reference/extensions/command_wrapper.rst

```diff
@@ -0,0 +1,33 @@
+.. _reference_extensions_command_wrapper:
+
+Command wrapper
+---------------
+
+The ``cmd_wrapper.py`` extension plugin is a Python scripts that receive the command line
+argument provided by ``self.run()`` recipe calls, and allows intercepting them and returning
+a new one. 
+
+This plugin must be located in the ``extensions/plugins`` cache folder, and can be installed
+with the ``conan config install`` command.
```

Comment:
If the others can also be installed via conan config I would also mention it

Reply:
What do you mean?

Reply:
Sorry, nevermind, I was thinking about the commands and deploy folders, but they are not explained here.